README.md

```diff
@@ -81,7 +87,152 @@ credentials for third-party services manually in order to run the development
 server or the integration tests.
 
 The environment variables in `.env` customize the settings from
-`opencraft/settings.py` which are loaded via `env()`.
+`opencraft/settings.py` which are loaded via `env()`. For more information about
+each setting, see [`opencraft/settings.py`](opencraft/settings.py). As a
```

Comment:
@omarkhan It's worth referencing the excellent settings documentation you created too, e.g.:

 "For more information about each setting, see [settings.py], or the [Application Settings](#application-settings) section below."